cmake/Findudev.cmake

```diff
@@ -60,7 +60,7 @@ find_path(UDEV_INCLUDE_DIR
 	)
 
 include(FindPackageHandleStandardArgs)
-find_package_handle_standard_args(UDEV
+find_package_handle_standard_args(udev
```

Comment:
Does the line below need to be changed to

```cmake
if(udev_FOUND)
```
?

Reply:
I have had some time to read the cmake doco. But it is not clear. Looking at this commit - it seems as if the correct syntax is UDEV_FOUND - https://github.com/rpavlik/cmake-modules/commit/3ce478298b438ec4bb80af87d7ac10ae4dcb5429

Reply:
Good enough for me!